scripts/get-envoy.bash

```diff
@@ -5,7 +5,7 @@ PATH="$PATH:$(go env GOPATH)/bin"
 export PATH
 
 _project_root="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)/.."
-_envoy_version=1.28.0
+_envoy_version=1.28.2
```

Comment:
I think we'll need to follow the pattern Caleb used here: https://github.com/pomerium/pomerium/pull/4897/files